src/v2/Apps/Conversation/Components/Message.tsx

```diff
@@ -102,14 +102,11 @@ export const Attachment: React.FC<AttachmentProps> = props => {
 }
 interface MessageProps extends Omit<BoxProps, "color"> {
   message: Message_message
-  initialMessage?: string
-  isFirst: boolean
   showTimeSince?: boolean
 }
 export const Message: React.FC<MessageProps> = props => {
-  const { message, initialMessage, isFirst, showTimeSince, ...boxProps } = props
+  const { message, showTimeSince, ...boxProps } = props
   const { isFromUser, body } = message
-  const text = isFirst ? initialMessage : body
```

Comment:
**praise:** This is a good follow up! I really should've removed it after https://github.com/artsy/metaphysics/pull/2326, but I'm glad you did so. Nice catch.